workflow_imaging/ingest.py

```diff
@@ -31,8 +31,8 @@ def ingest_sessions():
         sess_dir = pathlib.Path(session['session_dir'])
         acq_software = None
         recording_time = None
-        # search for either ScanImage or ScanBox files (in that order)
-        for scan_pattern, scan_type in zip(['*.tif', '*.sbx'], ['ScanImage', 'ScanBox']):
+        # search for either ScanImage, ScanBox, or Miniscope-DAQ files (in that order)
+        for scan_pattern, scan_type in zip(['*.tif', '*.sbx', '*.json'], ['ScanImage', 'ScanBox', 'Miniscope-DAQ']):
```

Comment:
relying on `*.json` alone is quite dangerous as this could match many other things given how widely used json format are for storing meta or config information

Reply:
I'd suggest to match by both the `*ms.mat` and check for the json later, or match by both the .mat and the json, or some combination in between